mapzen-places-api/src/test/java/com/mapzen/places/api/internal/PlaceAutocompletePresenterTest.java

```diff
@@ -49,6 +55,31 @@ public void getBoundingBox_shouldReturnCorrectBox() throws Exception {
   }
 
   @Test
+  public void getBoundingBox_lostClient_shouldReturnCorrectBox() throws Exception {
+    FusedLocationProviderService.FusedLocationProviderBinder stubBinder =
+          mock(FusedLocationProviderService.FusedLocationProviderBinder.class);
+    FusedLocationProviderService mockService = mock(FusedLocationProviderService.class);
+      when(stubBinder.getService()).thenReturn(mockService);
+    FusedLocationProviderApiImpl impl =
+        (FusedLocationProviderApiImpl) LocationServices.FusedLocationApi;
+    impl.onServiceConnected(stubBinder);
```

Comment:
Yeah its definitely a lot of work...I created a ticket to track this work https://github.com/mapzen/lost/issues/157